Common/src/CConfig.cpp

```diff
@@ -5175,6 +5183,35 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
       nTurbVar = 2; break;
   }
 
+  /*--- Postprocess SST_OPTIONS into structure. ---*/
+  //TODO this wont work.
+  string sst_string = "SST";
+  struct SST_Struct {
+
+    const Version;
+    const Modified;
+    const Production;
+    const Sustaining;
+    const Curvature;
+    //SAMPLE FAILURE LOGIC
+
+    //VERSION INCOMPATIBILITY
+    if (SST_Options_Map.find("V1994")) sst_string += "1994";
+    //TODO....
+
+    if (SST_Options_Map.find("V1994") && SST_Option_Map.find("V2003"))
+      SU2_MPI::Error(string("Two versions (1994 and 2003) selected for SST Options. Please choose only one."), CURRENT_FUNCTION);
+
+    //PRODUCTION INCOMPABILITY
+    unsigned short counter = 0;
+    if (SST_Options.Map_find("VORTICITY")) counter += 1;
+    if (SST_Options.Map_find("KATO-LAUNDER")) counter += 1;
+    if (SST_Options.Map_find("UQ")) counter += 1;
+    if (counter > 1) SU2_MPI::Error(string("Please select only one SST production term modifier (V, KL, UQ)."), CURRENT_FUNCTION);
+
+  };
+  //if (SST_Options_Map.find("VORTICITY")) cout <<"IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII"<<endl;
```

Comment:
This doesnt seem to work.

Reply:
The map is a static "look-up" to convert user strings into enum options, the options (enums) are read into SST_Options, not into SST_Options_Map.
So what you need is:
```c++
const auto sst_options_end = SST_Options + nSST_Options;
const bool sst_v = std::find(SST_Options, sst_options_end, SST_OPTIONS::VORTICITY) != sst_options_end;
```